lib/new_note_screen/day_in_one_sentence_widget.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+class DayInOneSentenceWidget extends StatelessWidget {
+  const DayInOneSentenceWidget({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(16),
+      margin: EdgeInsets.symmetric(horizontal: 16.0),
+      // height: 110,
+      // height: 150,
+      // width: 328,
+      decoration: BoxDecoration(
+        color: const Color(0xFFFFFFFE),
+        borderRadius: BorderRadius.circular(16.0),
+        boxShadow: [
+          BoxShadow(
+            color: const Color(0xFF738C93).withOpacity(0.03),
+            spreadRadius: 1,
+            blurRadius: 15,
+            offset: const Offset(0, 0),
+          ),
+        ],
+      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          const Text(
+            'Day in one sentence',
+            style: TextStyle(
+              fontSize: 16,
+              fontWeight: FontWeight.bold,
+            ),
+          ),
+          SizedBox(
+            height: 16,
+          ),
+          TextField(
+            keyboardType: TextInputType.multiline,
+            // maxLines: null,
+            inputFormatters: [
+              LengthLimitingTextInputFormatter(2000),
```

Comment:
add private const variable for 2000 value with understandable name. E.g _maxCharacterLength or something similar

Reply:
done